sdk/containerregistry/Microsoft.Azure.ContainerRegistry/src/Customizations/ContainerRegistryCredentials.cs

```diff
@@ -54,6 +54,8 @@ public enum LoginMode
         // Internal simplified client for Token Acquisition
         private ContainerRegistryRefreshToken _acrRefresh;
         private AuthToken _aadAccess;
+        private const string pattern = "\".+:.+:.+\"";  //Pattern to get the scope from headers
+        private static readonly Regex regex = new Regex(pattern);
```

Comment:
Either document what this is, or just rename to something more descriptive. For instance, `scopeFromHeaderRegex`